docs/ramp-up-plan.md

```diff
@@ -0,0 +1,67 @@
+
+Now would be the appropriate time to initiate a discussion with your security team to understand what security measures you need to put in place to train a FL model on sensitive data. This will help you understand the constraints you will have to work with, and will help set you up for success in the next phase.
+
+### Science track
+
+After that, you are ready to start developing the training procedure for your business problem. Our repository proposes some [industry-relevant examples](./README.md#real-world-examples). If your use case is not too different from one of those examples, that would be a good starting point! If you would like to see a new industry-relevant example similar to your use case added to our repository, please [open a Feature Request](https://github.com/Azure-Samples/azure-ml-federated-learning/issues/new?assignees=&labels=&template=feature_request.md&title=) and we will consider enriching our corpus of examples.
+
+Once you have trained your FL model, we recommend comparing it to a baseline model trained on the central data (or whatever other data allow for a sensible comparison). This will help you understand the impact of FL on your business problem, and assess whether you want to move forward.
+
+>**:checkered_flag: :walking: Checkpoint:** You have trained your first FL model relevant to your business goals and have demonstrated the feasibility of the approach and the value for your organization.
+
+## III. :running: **Run** - Provision a custom and _secure_ setup, connect to sensitive data, train real FL model
+
+This will likely apply to an organization that has validated its first FL prototype on synthetic/non-sensitive data and is ready to train a real FL model on production data. It is assumed that the Science part (how to train the model) has been mostly figured out at this point. 
+
+During this phase, you will be training a model on real, sensitive data, in a _secure_ setup. The focus will be on infrastructure.
+
+First, if you haven't already, now is time to create a setup leveraging VNets and private endpoints for added security. Again, please refer to our [provisioning README](./provisioning/README.md) to identify which ingredients you will need. Note that VNets and private endpoints  are a step in the right direction, but they are NOT silver bullets that guarantee 100% security. For instance, you will want to triple check your code to make sure data are not exposed by accident, or implement privacy-preserving algorithms such as Secure Aggregation or Differential Privacy. We will add instructions about these to our repository, so stay tuned!
+
+After that, you should connect your _sensitive_ data to your workspace. Depending on your use case, you have several options.
```

Comment:
note to self: need to clarify the scenario brought up by Chengyu (AKS compute in an external Azure tenant, data in storage account in same tenant)